packages/agoric-cli/src/commands/inter.js

```diff
@@ -235,34 +240,105 @@ export const makeInterCommand = (
     }
   };
 
-  const liquidationCmd = interCmd
-    .command('liquidation')
-    .description('liquidation commands');
-  liquidationCmd
+  const auctionCmd = interCmd
+    .command('auction')
+    .description('auction commands');
+  auctionCmd
     .command('status')
     .description(
-      `show amount liquidating, vault manager price
+      `show auction status in JSON format
 
 For example:
 
+inter auction status
 {
-  "liquidatingCollateral": "10 IbcATOM",
-  "liquidatingDebt": "120 IST",
-  "price": "12.00 IST/IbcATOM"
+  "schedule": {
+    "activeStartTime": null,
+    "nextStartTime": "2023-04-19T01:20:02.000Z",
+    "nextDescendingStepTime": "2023-04-19T01:20:02.000Z"
+  },
+  "book0": {
+    "startPrice": "9.99 IST/IbcATOM",
+    "startProceedsGoal": null,
+    "proceedsRaised": null,
+    "startCollateral": "0 IbcATOM",
+    "collateralAvailable": "0 IbcATOM",
+    "currentPriceLevel": "4.995 IST/IbcATOM"
+  },
+  "params": {
+    "DiscountStep": "5.00%",
+    "ClockStep": "00:00:10",
+    "LowestRate": "45.00%"
+  }
 }
 `,
     )
-    .option('--manager <number>', 'Vault Manager', Number, 0)
-    .action(async opts => {
-      const { agoricNames, readLatestHead } = await tryMakeUtils();
-
-      const [metrics, quote] = await Promise.all([
-        readLatestHead(`published.vaultFactory.manager${opts.manager}.metrics`),
-        readLatestHead(`published.vaultFactory.manager${opts.manager}.quotes`),
-      ]);
-      const info = fmtMetrics(metrics, quote, values(agoricNames.vbankAsset));
-      show(info, true);
-    });
+    .option('--book <number>', 'Auction Book', Number, 0)
+    .action(
+      async (
+        /**
+         * @type {{
+         *   book: number,
+         * }}
+         */ opts,
+      ) => {
+        const { agoricNames, readLatestHead } = await tryMakeUtils();
+
+        /**
+         * ref publishSchedule()
+         * packages/inter-protocol/src/auction/scheduler.js#L103
```

Comment:
this line number will be wrong soon enough that it should be omitted. Same for the source path less urgently. For a durable ref try,
```suggestion
         * @see (import('@agoric/inter-protocol/src/auction/scheduler.js')} publishSchedule
```
I don't think our tools error on bad paths there yet but they will

Reply:
oh but more importantly… none of these types should be defined here. The contract must declare its published types and it does.
https://github.com/Agoric/agoric-sdk/blob/470ca4121d46170665958cd713ec9d22dac5eb9a/packages/inter-protocol/src/auction/scheduler.js#L43-L52
https://github.com/Agoric/agoric-sdk/blob/470ca4121d46170665958cd713ec9d22dac5eb9a/packages/inter-protocol/src/auction/auctionBook.js#L93-L107

Reply:
Thanks! I did just a tiny bit of looking for them, but didn't see them when I hovered over the point of publication.

Reply:
yeah there's some type flow regression in `makeRecorderKit`. I'll look into it.

Reply:
fix in 68d4170d38622089ac3b24f5bf40c1fd0783dff9

Reply:
ok, I applied that in this PR. Didn't manage to preserve authorship, though.